config/i18n-tasks.yml

```diff
@@ -97,6 +97,7 @@ ignore_unused:
   - 'experiments.*'
   - 'event_types.*'
   - 'zxcvbn.*'
+  - 'links.cancel'
```

Comment:
Correction: the `links.cancel` entry is still present in `config/locales/links/en.yml`, but it's not triggering an error.